### ISSUE TITLE
[Keyboard Navigation – Make Code - Share] Keyboard focus indicator is not visible on 'Publish project' button under 'Share Project' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3590]
**User Experience**:
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of  "Share Project" dialog.
5. Verify that keyboard focus indicator is visible on 'Publish Project' button or not.

**Actual Result:**
Keyboard focus indicator is not visible on 'Publish project' button under 'Share Project' pop-up window and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on 'Publish project' button under 'Share Project' pop-up window, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.4.7 - Focus Visible
](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.7, #Win10-Edge(Chromium)

[WCAG2.4.7_Share Project_Focus is not visible.zip](https://github.com/microsoft/pxt-microbit/files/5493139/WCAG2.4.7_Share.Project_Focus.is.not.visible.zip)

**Comment by jwunderl** [comment]
focus visible with color change, outline in high contrast
